gpustat/core.py

```diff
@@ -299,10 +319,14 @@ def get_process_info(nv_process):
                 if not _cmdline:
                     # sometimes, zombie or unknown (e.g. [kworker/8:2H])
                     process['command'] = '?'
+                    process['full_command'] = '?'
                 else:
                     process['command'] = os.path.basename(_cmdline[0])
+                    process['full_command'] = " ".join(_cmdline)
                 # Bytes to MBytes
                 process['gpu_memory_usage'] = nv_process.usedGpuMemory // MB
+                process['cpu_percent'] = ps_process.cpu_percent(interval=0.1)
+                process['cpu_memory_usage'] = ps_process.memory_percent()
```

Comment:
I think showing Memory usage in MB/GB (human readable) rather than percentage would be more helpful.

Reply:
Good idea. I added this.